contracts/account-recovery/AAccountRecovery.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.21;
+
+import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+import {IAccountRecovery} from "../interfaces/account-recovery/IAccountRecovery.sol";
+import {IRecoveryProvider} from "../interfaces/account-recovery/IRecoveryProvider.sol";
+
+/**
+ * @notice The Account Recovery module
```

Comment:
Can you please describe that this is an auxiliary helper for EIP-7947 + add the link to the EIP.